apps/user/tests/test_password.py

```diff
@@ -0,0 +1,17 @@
+from django.core.exceptions import ValidationError
+
+from deep.tests import TestCase
+
+from user.validators import CustomMaximumLengthValidator
+
+
+class PasswordCheckerTest(TestCase):
+
+    def test_password_greater_than_128_characters(self):
+        expected_error = "This password has exceed the limit of %d characters"
+        self.assertIsNone(CustomMaximumLengthValidator().validate('12345678'))
+        self.assertIsNone(CustomMaximumLengthValidator(max_length=20).validate('123'))
+
+        with self.assertRaises(ValidationError) as vd:
+            CustomMaximumLengthValidator(max_length=128).validate('12'*129)
+        self.assertEqual(vd.exception.messages, [expected_error % 128])
```

Comment:
Message can change with time. Lets check against `exception.code` instead.

Reply:
@k9845 is referencing from this test https://github.com/django/django/blob/5fcfe5361e5b8c9738b1ee4c1e9a6f293a7dda40/tests/auth_tests/test_validators.py#L98-L117

Yeah, the message can change, so the code sounds good.